assets/javascripts/discourse/helpers/category-expert-question-indicator.js

```diff
@@ -7,7 +7,7 @@ export function categoryExpertQuestionIndicator(topic, currentUser) {
 
   if (
     currentUser.staff ||
-    topic.creator.id === currentUser.id ||
+    topic.creator?.id === currentUser.id ||
```

Comment:
Is our babel currently configured to transpile this operator?

Reply:
I was under the impression that it didn't need to be transpiled 
![image](https://user-images.githubusercontent.com/16214023/112885393-52962d80-9096-11eb-9040-17555105a402.png)

Reply:
Oh nice, we can use this chaining operator in a lot of places.